source/images/download-hover.svg

```diff
@@ -0,0 +1 @@
+<svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><title>download-hover</title><path d="M17 8h-4V2H7v6H3l7 7 7-7zM3 17v2h14v-2H3z" fill="#0288D1" fill-rule="evenodd"/></svg>
```

Comment:
Would probably be better to put the hover state within the SVG icon files rather than swapping images.